disperser/dataapi/operator_handler.go

```diff
@@ -95,7 +95,11 @@ func (oh *OperatorHandler) ProbeV2OperatorPorts(ctx context.Context, operatorId
 		return &OperatorPortCheckResponse{}, err
 	}
 
-	operatorSocket := core.OperatorSocket(operatorInfo.Socket)
+	operatorSocket, err := core.ParseOperatorSocket(operatorInfo.Socket)
+	if err != nil {
+		oh.logger.Warn("failed to parse operator socket", "operatorId", operatorId, "error", err)
```

Comment:
In what cases it'll fail to parse? Is that just a warning issue?

Reply:
Here and below: useful to wrap this error with operatorId information, so it's more clear what went wrong